playbooks/roles/gitea/templates/app.ini.j2

```diff
@@ -345,9 +345,14 @@ ENABLE_AUTO_REGISTRATION = true
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;[mailer]
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+[mailer]
+ENABLED        = true
```

Comment:
maybe we should wrap this into at least "{% if gitea_smtp_host is defined %}"? And then maybe make type and tls configurable as well (with defaults)?